extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -14,5 +14,13 @@
  * limitations under the License.
  */
 
- amp-sticky-ad {
- }
+amp-sticky-ad {
+  position: fixed !important;
+  text-align:center;
+  bottom: 0 !important;
+  left: 0;
+  z-index: 1;
+  width: 100%;
+  /* TODO(zhouyx): discuss on what the max-height should be? */
```

Comment:
have a bug open for this and link it here - explain the problem well on the bug.